frontend/src/redux/reducers/ui.tsx

```diff
@@ -180,6 +181,12 @@ function reducer(state = _.cloneDeep(INITIAL_STATE), action: Action) {
       setTheme(newFilters.theme.name);
       break;
     }
+    case UI_RESET_PLUGIN_VIEWS: {
+      const newState = _.cloneDeep(INITIAL_STATE);
```

Comment:
can we just do 
`const newState = {...INITIAL_STATE};`

Reply:
That doesn't clone deeper levels and thus may touch existing objects.